docs/features/basic-concepts/fee.md

```diff
@@ -5,30 +5,34 @@ Specify the maximum gas limit by `--gas`.
 If maximum gas is too small, it won't be enough for executing the transaction. 
 If the fee is too low, fee paid for each unit of gas will be less than gaslimit and 
 validators won't execute the transaction neither. 
-The fee(minimum unit)/gas must be large than 6*10^12. 
+The fee(minimum unit)/gas must be large than 6*10^12 iris-atto. 
 We recommend that you set your maximum gas to 50000 and set your maximum fee to 0.3iris. 
 Fee will be consumed according to actual gas used and spare fee will be reimbursed to users.
 
+> Note: Individual transactions consume more gas (e.g. creating validators), so it is recommended to set `--gas=100000 --fee=0.6iris` to ensure the smooth execution of transactions.
+
 ## Fee
 
-To secure their own validator node and maintain the avalibility of blockchain network, 
+To secure their own validator node and maintain the availability of blockchain network, 
 validators in IRISnet need a lot of equipments and resources. 
 Thus, every transactions in IRISnet should pay some fee to validators. 
 The parameter in commands is used to specify the maximum fee the user want to pay for their transaction.
 
 ## Gas
 
-The resource needed for every transactions are varied for different type of transactions. For example, only a few computations, queries & modifies is needed for sending some token to other peolpe. But a lot of computations, queries & modifies is needed for creating a validator.  Gas is the unit used to measure how much resource needed to execute the transaction. We list the gas needed for some typical operactions in the below:
+The resource needed for every transactions are varied for different type of transactions. For example, only a few computations, queries & modifies are needed for transfer transaction. But more computations, queries & modifies are needed for creating a validator.  Gas is the unit used to measure how much resource needed to execute the transaction. We list the gas needed for some typical operations in the below:
 
-- gas needed  for writing the tx to the database is: 10 * the size of the transaction data (in bytes)
+- gas needed for writing the tx to the database is: 10 * the size of the transaction data (in bytes)
 - gas needed for reading some data from database: 10 + data length(in bytes)
 - gas needed for writing some data to database: 10 + 10 * data length(in bytes)
-- sign or verify a signature
+- sign or verify a signature : 100
+- ....
```

Comment:
Maybe a table for these gas consumption is better

Reply:
Good suggestion

Reply:
fixed